### PR TITLE
add https redirection

### PR DESCRIPTION
**Pull request by RamyChaabane**
force https redirection when `http_option = redirected`

### CODE REVIEW

File: tests/functions/functions.test.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ const { expect } = require('chai');
 const { expect: jestExpect, it } = require('@jest/globals');
 
 const { getTmpDirPath, replaceTextInFile } = require('../utils/fs');
-const { getServiceName, sleep, serverlessDeploy, serverlessRemove} = require('../utils/misc');
```
Comment (RamyChaabane) [inline tests/functions/functions.test.js:9]: IDE was signalling a text convention error, so I had to break it down into two lines

File: README.md [hunk 2]
```diff
@@ -158,6 +158,7 @@ The configuration includes the following parameters:
   * `runtime`: (Optional) runtime of the function, if you need to deploy multiple functions with different runtimes in your Serverless Project. If absent, `provider.runtime` will be used to deploy the function, see [this example project](./examples/multiple).
   * `events` (Optional): List of events to trigger your functions (e.g, trigger a function based on a schedule with `CRONJobs`). See `events` section below
   * `custom_domains` (Optional): List of custom domains, refer to the [how to add a custom domain](https://www.scaleway.com/en/docs/compute/functions/how-to/add-a-custom-domain-name-to-a-function/) documentation page
+  * `httpOption` (Optional): force https redirection, possible values are `enabled` and `redirected` (default: `enabled`)
```
Comment (thomas-tacquet) [inline README.md:161]: Do we have somewhere in Scaleway documentation where the httpOption behaviour is explained ? If sou can you add a link here ?

Comment (RamyChaabane) [reply README.md:161]: I've checked with Lucas, no there isn't

File: tests/functions/functions.test.js [hunk 3]
```diff
@@ -18,6 +19,7 @@ const serverlessExec = path.join('serverless');
 const stringIdentifier = '# second-function-identifier';
 const serverlessFile = 'serverless.yml';
 const descriptionTest = 'slsfw test description';
+const httpOptionTest = 'redirected';
```
[the remark this answers is not in the document]
Comment (RamyChaabane) [reply tests/functions/functions.test.js]: done

**Review by thomas-tacquet — APPROVED** [review]

**Pull request merged**